cns/client_test.go

```diff
@@ -140,6 +143,17 @@ func TestClient(t *testing.T) {
 			},
 		},
 	}
+
+	isvSphereVersion91orAbove := isvSphereVersion91orAbove(ctx, c.ServiceContent.About)
+
+	var generatedVolumeUUIDFromClient string
+	if isvSphereVersion91orAbove {
+		generatedVolumeUUIDFromClient = uuid.New().String()
```

Comment:
We don't need this uuid.NewString() in tests. We can pass "901e87eb-c2bd-11e9-806f-005056a0c9a0" which is the pvc uid from Line 134